packages/babel-plugin-proposal-do-expressions/test/fixtures/do-expressions/labeled-break/input.js

```diff
@@ -0,0 +1,6 @@
+const x = do {
+  y: {
+    'out';
+    break y;
```

Comment:
We can not remove labelled break statement without more execution context, i.e.
```js
// returns 42
do { y: { "out"; break y; } 42 }
```
I don't come up with a satisfiable fix regarding to the LabeledStatement without significant rewriting `getCompetionsRecords`. I suggest we separate this fix into another PR and this PR is focused on SwitchStatement.

Reply:
Will do, but does `getCompletionRecords` not get the last node of the `BlockStatement`, which would be `42` in this case?

Reply:
If we replace `"out"; break y;` by `return "out"`, the compiled output will be
```js
(() => { y: { return "out"; } return 42 } })()
// returns "out"
```
I don't think we correctly handle the labeled break statement in `getCompletionRecords`, so I suggest we address `LabelStatement` in another PR.

Reply:
Oh I see. I forgot about the loop. Thanks for the feedback, and sounds good!